app/views/qsets/_question_page_header_large_screen.html.erb

```diff
@@ -12,5 +12,6 @@
 <div class="row col-xs-12">
   <%= render "breadcrumbs" %>
 </div>
-
-
+<div class=" row col-xs-12 center">
```

Comment:
Why isn't this like the _question_page_header_small_screen version? Can they not be the same? If they have to be different, we shouldn't use `.row` and `.col-...` in the same element, they need to be nested, see [bootstrap docs](http://getbootstrap.com/css/#grid-intro) (points 2 and 3). Also a nitpick: extra space before `row` class.